src/core/Accordion/Accordion.stories.tsx

```diff
@@ -204,3 +209,18 @@ export const StaticAndFullyOpen = {
     },
   },
 };
+
+export const CustomClickableButton = {
+  render: () =>
+    AccordionPresentation({
+      data: dataWithCustomClickable,
+    }),
+  parameters: {
+    docs: {
+      description: {
+        story:
+          "Supplying an `onClick` function to a section adds a clickable function. This can be useful for custom actions.",
```

Comment:
Adds a clickable function to what? Forgive me if I'm wrong but this sounds Copilotted - it'd be better to have a more descriptive sentence here to describe the purpose of what this change does.

Reply:
A reminder that we should only accept Copilot's suggestions when we fully agree with and understand what it suggests